src/stratigraphy/extract.py

```
@@ -2,11 +2,13 @@
 
 import logging
 import math
+from typing import List, Tuple
```

Ruff

`typing.List` is deprecated, use `list` instead

(UP035)